src/elements/flip-card/flip-card/flip-card.ts

```diff
@@ -0,0 +1,74 @@
+import type { CSSResultGroup, TemplateResult } from 'lit';
+import { html, LitElement } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+
+import { SbbLanguageController } from '../../core/controllers.js';
+import { i18nFlipCard, i18nReverseCard } from '../../core/i18n.js';
+import '../../button/secondary-button.js';
+import type { SbbFlipCardDetailsElement } from '../flip-card-details.js';
+import type { SbbFlipCardSummaryElement } from '../flip-card-summary.js';
+
+import style from './flip-card.scss?lit&inline';
+
+/**
+ * Displays an informative card that reveals more informations upon being clicked.
+ *
+ * @slot summary - Use this slot to provide a sbb-flip-card-summary component.
+ * @slot details - Use this slot to provide a sbb-flip-card-details component.
+ *
+ */
+@customElement('sbb-flip-card')
+export class SbbFlipCardElement extends LitElement {
```

Comment:
Due to the slotchange event being used and chaning state of children, we should use the hydration mixin here.